backend/app/domains/courses/crud.py

```
@@ -1,21 +1,32 @@
+from app.domains.registrations.crud import delete_registrations_by_course_id
 from app.domains.registrations.models import Register
 from app.domains.students.models import Student
+from app.domains.users.models import User, UserRole
 
 
 def get_courses(db: Session) -> list[Course]:
     return paginate(db, select(Course))
 
 
-def create_course(db: Session, course_in: CourseCreate) -> Course:
+def create_course(db: Session, course_in: CourseCreate, current_user: User) -> Course:
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)